pkg/client/unversioned/remotecommand/v1.go

```diff
@@ -40,7 +41,7 @@ type streamProtocolV1 struct {
 
 var _ streamProtocolHandler = &streamProtocolV1{}
 
-func (e *streamProtocolV1) stream(conn httpstream.Connection) error {
+func (e *streamProtocolV1) stream(conn httpstream.Connection, _ <-chan term.Size) error {
```

Comment:
would hide this behind an interface - raw channels may not always solve the problem.